schema.go

```diff
@@ -768,6 +769,19 @@ func parseTimeUnit(arg string) (TimeUnit, error) {
 	return nil, fmt.Errorf("unknown time unit: %s", arg)
 }
 
+func parseUTCNormalization(arg string) (isUTCNormalized bool, err error) {
+	switch arg {
+	case "utc":
+		return true, nil
+	case "local":
+		return false, nil
+	default:
+
+	}
+
+	return false, fmt.Errorf("unknown utc normalization: %s", arg)
```

Comment:
Moving the return in the default for style and because the default was doing nothing:
```suggestion
	default:
		return false, fmt.Errorf("unknown utc normalization: %s", arg)
	}
```